packages/manager/modules/pci/src/projects/project/storages/objects/object/object.routing.js

```diff
@@ -1,16 +1,23 @@
 export default /* @ngInject */ ($stateProvider) => {
   $stateProvider.state('pci.projects.project.storages.objects.object', {
-    url: '/{containerId}',
+    url: '/{containerId}/{isHighPerfStorage}',
```

Comment:
Do we expect to have `https://www.ovh.com/manager/public-cloud/xxx/true` in the url path ? Should it be transformed as `/{containerId}?isHighPerfStorage` ?